json_schema/project.schema.json

```diff
@@ -0,0 +1,310 @@
+            "type": "string",
+            "description": "Relation of this Media Item.",
+            "enum": [
+              "icon",
+              "gallery"
+            ]
+          },
+          "type": {
+            "type": "string",
+            "description": "The type of this Media Item.",
+            "enum": [
+              "image",
+              "video"
+            ]
+          },
+          "src": {
+            "type": "string",
+            "description": "The source of this Media Item.",
+            "format": "uri"
+          },
+          "caption": {
+            "type": "string",
+            "description": "A caption for this Media Item. Cannot contain HTML."
+          },
+          "sha256": {
+            "type": "string",
+            "description": "An SHA256 hash of the item linked to in src. Conflicts with embed."
+          },
+          "embed": {
+            "type": "string",
+            "description": "A link to an embedded webpage. Conflicts with sha256 and src.",
+            "format": "uri"
+          }
+        },
+        "oneOf": [
+          {
+            "properties": {
+              "embed": false
+            },
+            "required": [
+              "src",
+              "rel",
+              "type"
+            ]
+          },
+          {
+            "properties": {
+              "sha256": false,
+              "src": false
+            },
+            "required": [
+              "rel",
+              "type",
+              "embed"
+            ]
+          }
+        ]
+      }
+    },
+    "authors": {
+      "type": "array",
+      "description": "The authors of this Project.",
+      "items": {
+        "oneOf": [
+          {
+            "type": "object",
+            "description": "An individual author.",
+            "$ref": "#/definitions/author"
+          },
+          {
+            "type": "object",
+            "description": "A Team consisting of multiple authors.",
+            "additionalProperties": false,
+            "required": [
+              "team",
+              "id",
+              "name",
+              "members"
+            ],
+            "properties": {
+              "team": {
+                "const": true,
+                "description": "Indicates whether or not this is a Team. Must be 'true' for Teams."
+              },
+              "id": {
+                "type": "string",
+                "description": "A unique identifier of this Team."
+              },
+              "name": {
+                "type": "string",
+                "description": "A human-readable name of this Team."
+              },
+              "uri": {
+                "type": "string",
+                "description": "A URI users can visit to view more about this Team.",
+                "format": "uri"
+              },
+              "members": {
+                "type": "array",
+                "description": "A list of authors that are part of this t Team.",
```

Comment:
There is a lost "t" in the value string.

Reply:
My bad.